lib/config.coffee

```diff
@@ -74,7 +74,7 @@ exports.generate = (options, params = {}) ->
 		appUpdatePollInterval: params.appUpdatePollInterval or 60000
 
 		listenPort: 48484
-		vpnPort: options.vpnPort or 1723
+		vpnPort: _.parseInt(options.vpnPort) or 1723
```

Comment:
is there any reason to use _.parseInt here?

there is a built-in parseInt in javascript